### PR TITLE
Inline stats views

### PR DESCRIPTION
**Pull request by phlax**
Adds inline stats to views

### CODE REVIEW

File: pootle/core/views/browse.py [hunk 1]
```diff
@@ -104,35 +142,26 @@ def get_context_data(self, *args, **kwargs):
...
-            limit=TOP_CONTRIBUTORS_CHUNK_SIZE + 1,
-        )
-
+        table = self.table
+        stats = self.stats
+        del stats["children"]
         ctx.update(
             {'page': 'browse',
-             'stats_refresh_attempts_count':
-                 settings.POOTLE_STATS_REFRESH_ATTEMPTS_COUNT,
```
Comment (phlax) [inline pootle/core/views/browse.py]: yep - not needed

File: pootle/core/views/browse.py [hunk 2]
```diff
@@ -104,35 +142,26 @@ def get_context_data(self, *args, **kwargs):
...
              'can_translate': can_translate,
              'can_translate_stats': can_translate_stats,
              'url_action_continue': url_action_continue,
              'url_action_fixcritical': url_action_fixcritical,
              'url_action_review': url_action_review,
              'url_action_view_all': url_action_view_all,
-             'table': self.table,
+             'top_scorers': self.top_scorers,
+             'top_scorers_data': self.top_scorer_data,
+             'table': table,
```
Comment (phlax) [inline pootle/core/views/browse.py]: it has to be done this way has `table` uses `self.stats["children"]`

File: pootle/core/views/browse.py [hunk 3]
```diff
@@ -57,30 +71,55 @@ def sidebar_announcements(self):
...
+        return (
+            {'id': self.table_id,
+             'fields': self.table_fields,
+             'headings': get_table_headings(self.table_fields),
+             'items': self.items,
+             'disabled_items': self.disabled_items}
+            if (self.table_id
+                and self.table_fields
+                and self.items)
+            else None)
```
Comment (phlax) [inline pootle/core/views/browse.py]: ill just revert this change - returning None is pointless

File: pootle/core/views/browse.py (hunk 2 above)
Comment (unho) [inline pootle/core/views/browse.py]: Please add a comment with the rationale to prevent this from being cleaned up.

**Comment by unho** [comment]
Nits aside, lgtm.

File: pootle/core/views/browse.py (hunk 2 above)
Comment (phlax) [inline pootle/core/views/browse.py]: added
